src/fides/api/custom_types.py

```diff
@@ -116,6 +116,20 @@ def validate_path_of_url(value: AnyUrl) -> str:
 
     We perform the basic URL validation, but also prevent URLs with paths,
     as paths are not part of an origin.
+
+    As this is meant to be used as a validator _after_ the AnyUrl validator,
+    we implicitly disallow `*` to be specified as an origin value, since it is
+    rejected by the AnyUrl validator.
+
+    Note that `*` _is_ considered a valid origin value
+    (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin),
+    but we specifically disallow it to be set on the `cors_origins` security setting
+    in our application. This is because non-owner users (contributors)
+    are able to edit the `cors_origins` security setting (via API), and we do not
+    want to allow them to set a wildcard origin.
+
+    Instead, `*` can be set via the `cors_origin_regex` security setting.
```

Comment:
```suggestion
    Instead, `.*` can be set via the `cors_origin_regex` security setting as an equivalent to `"*"`. There is an behavioral difference in that the `Access-Control-Allow-Origin` in the response will be set to the `Access-Control-Allow-Origin` in the request rather than `"*"`.
```

Reply:
thanks for the catch! put in something inspired by your suggestion in https://github.com/ethyca/fides/pull/5853/commits/e92d25369defb8b26537e2ca47b92a3c038107a8